templates/vm/guest.xml.j2

```diff
@@ -142,6 +142,13 @@
             <source bridge="{{ bridge.name }}"/>
             <mac address="{{ bridge.mac_address }}"/>
             <model type="virtio"/>
+            <virtualport type='openvswitch'/>
+            <address type='pci' domain='0x0000' bus='0x03' slot='0x01' function='0x0'/>
```

Comment:
is it a good idea to fix the pci allocation details?

Reply:
Nope, I removed them